resources/windows/wix/main.wxs.erb

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="windows-1252"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Product Id="<%= ProductCode %>" UpgradeCode="<%= UpgradeCode %>" Name="<% ProductName %>" Language="1033" Codepage="1252" Version="<%= ProductVersionNumber %>" Manufacturer="<%= OurCompanyName %>">
+    <Package InstallerVersion="300" InstallScope="perMachine" Description="<% ProductName %> Installer" Comments="http://www.puppetlabs.com/" Compressed="yes" Platform="<%= Platform %>" />
+    <MajorUpgrade AllowDowngrades="yes" />
+    <UI>
+      <UIRef Id="WixUI_PuppetInstallDir"/>
+      <Property Id="WIXUI_INSTALLDIR" Value="INSTALLDIR" />
+      <Property Id="WIXUI_EXITDIALOGOPTIONALTEXT" Value="Manage your first resources on this node, explore the Puppet community and get support using the shortcuts in the Documentation folder of your Start Menu." />
+    </UI>
+    <Media Id="1" Cabinet="<%= Product %>.cab" EmbedCab="yes" CompressionLevel="high" />
+
+    <Feature Id="<%= ProductName %>Runtime" Title="<%= OurProductName %> Runtime" Level="1">
+      <ComponentRef Id="ProductComponentGroup" />
+    </Feature>
+
+    <Directory Id='TARGETDIR' Name='SourceDir'>
+      <Directory Id="<%= PlatformProgramFilesFolder %>" >
+        <Directory Id='PuppetLabs' Name="<%= CompanyName %>">
+          <Directory Id='INSTALLDIR' Name="<%= OurProductName %>"/>
```

Comment:
What about if different projects aren't installing here? Or have a different number of nested directories for their `INSTALLDIR`?